types/src/v20/network.rs

```diff
@@ -0,0 +1,26 @@
+// SPDX-License-Identifier: CC0-1.0
+
+//! The JSON-RPC API for Bitcoin Core `v0.20.2` - network.
+//!
+//! Types for methods found under the `== Network ==` section of the API docs.
+
+use serde::{Deserialize, Serialize};
+
+/// Result of JSON-RPC method `listbanned`.
+///
+/// > listbanned
+///
+/// > List all banned IPs/Subnets.
+#[derive(Clone, Debug, PartialEq, Deserialize, Serialize)]
+pub struct ListBanned(pub Vec<Banned>);
```

Comment:
This is the same shape for all versions so I'd just use the v17 one everywhere and only add the new `Banned` type.

Reply:
oooph you are correct, my bad.